shoop/core/models/_contacts.py

```diff
@@ -359,7 +359,7 @@ def get_person_contact(user):
         'last_name': getattr(user, 'last_name', ''),
         'email': getattr(user, 'email', ''),
     }
-    return PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
+    return PersonContact.objects.filter(is_active=True).get_or_create(user=user, defaults=defaults)[0]
```

Comment:
I think it would better written as
```python
contact = PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
if contact.is_active:
    return contact
return None
```
And if we want to have that `allow_inactive`, then just add ` or allow_inactive` to the if. I don't know if it's needed, but wouldn't hurt though...